pkg/restapi/vault/controller.go

```diff
@@ -11,9 +11,14 @@ import (
 	"github.com/trustbloc/edge-service/pkg/restapi/vault/operation"
 )
 
+// Config describes operation`s configuration.
+type Config operation.Config
+
 // New returns new controller instance.
-func New(config *operation.Config) (*Controller, error) {
-	vaultService, err := operation.New(config)
+func New(config *Config) (*Controller, error) {
+	cfg := operation.Config(*config)
```

Comment:
nit: this is one reason why this `controller` package is useless

Reply:
Removed!